walmart/walmart.py

```diff
@@ -141,6 +153,9 @@ def get_payload(self, items):
                 xmlns='http://walmart.com/'
             )
         )
+        data = '--%s\n\n' % self.boundary + inv_data + \
```

Comment:
'--{boundary}\n\n{data}\n--{boundary}--'.format(boundary=self.boundary, data=inv_data)

^ format is the new elegant way but be careful though (http://lucumr.pocoo.org/2016/12/29/careful-with-str-format/)

Reply:
👍